airbyte-cdk/java/airbyte-cdk/db-sources/src/main/kotlin/io/airbyte/cdk/integrations/debezium/internals/DebeziumRecordIterator.kt

```diff
@@ -212,6 +233,11 @@ class DebeziumRecordIterator<T>(
         }
     }
 
+    private fun isTest(): Boolean {
+        val isTest = config.has("is_test") && config["is_test"].asBoolean()
+        return !isTest
+    }
```

Comment:
Did you mean this?

```
    private fun isTest(): Boolean {
        return config.has("is_test") && config["is_test"].asBoolean()
    }
```

Reply:
Ah yes - I had just ported this over from the previous function :(